app/src/main/java/com/quran/labs/androidquran/presenter/data/JuzListPresenter.kt

```diff
@@ -13,10 +13,11 @@ class JuzListPresenter @Inject constructor(
 
   suspend fun quarters(): List<String> {
     return withContext(Dispatchers.IO) {
-      val ayahIds = quranInfo.quarters
-        .map { quranInfo.getAyahId(it.first(), it.last()) }
+      val ayahIds = quranInfo.quarters.map { intArray ->
+        quranInfo.getAyahId(intArray.first(), intArray.last())
+      }
       val results = arabicDatabaseUtils.getAyahTextForAyat(ayahIds)
-      ayahIds.map { results[it] ?: "" }
+      ayahIds.map { i -> results[i] ?: "" }
```

Comment:
nitpick - prefer `it` here or `id` if you want to name it

Reply:
will change it to `id` in other PR 👍🏻